perllib/FixMyStreet/Script/Inactive.pm

```diff
@@ -0,0 +1,79 @@
+package FixMyStreet::Script::Inactive;
+
+use v5.14;
+use warnings;
```

Comment:
If this is just to turn on say then `use feature qw/say/` might be more explicit about it.

Reply:
Not just that! Also means I don't have to say `use strict`, has better (correct) Unicode handling on string operations. Could also use `state`. Probably should have it everywhere :)